folium/features.py

```diff
@@ -49,14 +49,18 @@ class WmsTileLayer(Layer):
         Adds the layer as an optional overlay (True) or the base layer (False).
     control : bool, default True
         Whether the Layer will be included in LayerControls
+    **kwargs : additional keyword arguments
+        Passed through to the underlying tileLayer.wms object and can be used
+        for setting extra tileLayer.wms parameters or as extra parameters in
+        the WMS request.
 
     For more information see:
     http://leafletjs.com/reference.html#tilelayer-wms
 
     """
     def __init__(self, url, name=None, layers=None, styles=None, fmt=None,
                  transparent=True, version='1.1.1', attr=None, overlay=True,
-                 control=True):
+                 control=True, **kwargs):
```

Comment:
:+1:  ideally we should validade those kw args before passing to the render b/c they fail silently. But that can be done in another PR. Let us assume that people adding extra options know what they are doing.